pkg/configfilearg/parser.go

```diff
@@ -46,6 +46,32 @@ func (p *Parser) Parse(args []string) ([]string, error) {
 	return args, nil
 }
 
+func (p *Parser) FindString(args []string, target string) (string, error) {
+	configFile, isSet := p.findConfigFileFlag(args)
+	if configFile != "" {
+		bytes, err := readConfigFileData(configFile)
+		if !isSet && os.IsNotExist(err) {
```

Comment:
This is a nit but the `if err != nil` should be first and within that, the isSet and os.IsNot... calls. So we don't have to check any of that if err is nil.

Reply:
This logic came from the function above it, I will address this in a separate PR after merging this one